skforecast/model_selection/model_selection.py

```diff
@@ -819,6 +819,7 @@ def grid_search_forecaster(
         Results for each combination of parameters.
 
         - column lags: lags configuration for each iteration.
+        - column lags_labels: Descriptive labels or aliases for the lags.
```

Comment:
@JavierEscobarOrtiz 
Should we mention that this is only used when lags_grid is a dict, and that keys as used as label?